rpm/build.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+# This script will package the `update` script into a `rpm` package.
+
+declare -r NAME="update"
+declare -r BUILD_DIR="$(pwd)"
+declare -r VERSION="$(grep "version=" "${BUILD_DIR}/${NAME}" | cut -d"\"" -f2 | cut -d"v" -f2)"
+
+# Exit out when running the script as root.
+if [[ ${UID} -eq 0 ]]; then
+   printf 'Please run this script as a normal user.'
+   exit 1
+fi
+
+# Install the necessary packages when needed.
+command -v rpmdev-setuptree &> /dev/null || sudo dnf install rpm-build rpmdevtools
```

Comment:
```suggestion
command -v rpmdev-setuptree &> /dev/null && command -v rpmbuild &>/dev/null || sudo ${PKGMANGER} install rpm-build rpmdevtools -y
```
The script failed to detect that `rpmbuild` wasn't installed, so it errored out. This should fix it. I also changed the package manager to be current with my other suggestion.